packages/wrangler/src/versions/upload.ts

```diff
@@ -384,6 +384,11 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 			placement,
 			tail_consumers: config.tail_consumers,
 			limits: config.limits,
+			annotations: {
+				"workers/message": props.message,
+				"workers/tag": props.tag,
+				"workers/triggered_by": "wrangler",
```

Comment:
Can we remove triggered_by while we're here? As this will no longer be overridable soon.